### DISCUSSION TITLE
Could I use web3auth in solana-wallet-adapter ?

### DISCUSSION THREAD

**Discussion by ieow** [Web3Auth/Web3Auth#311] [Help]
Could I use web3auth in solana-wallet-adapter ?

**Comment by himanshuchawla009** [comment]
So Web3Auth is not a wallet, but it provides infra to build wallets on top of it. Currently, Solana wallet (https://solana.tor.us) which is built using Web3Auth is supported with solana-wallet-adapter. Similarly, you can build your own wallet with Web3Auth which can be added to  solana-wallet-adapter.
